tests/Doctrine/Tests/DBAL/Functional/Ticket/GH3423Test.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+use Doctrine\Tests\DbalFunctionalTestCase;
+use function in_array;
+
+/**
+ * @see https://github.com/doctrine/dbal/issues/3423
+ */
+class GH3423Test extends DbalFunctionalTestCase
```

Comment:
Please avoid using issue numbers in tests. It should be named after the feature/behavior being tested.